qa/integration/services/helpers.sh

```diff
@@ -13,14 +13,29 @@ setup_install_dir() {
 
 wait_for_port() {
     count=$PORT_WAIT_COUNT
-    port=$1
-    while ! nc -z localhost $port && [[ $count -ne 0 ]]; do
+    while ! test_port "$1"  && [[ $count -ne 0 ]]; do
         count=$(( $count - 1 ))
         [[ $count -eq 0 ]] && return 1
         sleep 0.5
     done
     # just in case, one more time
-    nc -z localhost $port
+    test_port "$1"
+}
+
+test_port() {
+    if command -v nc 2>/dev/null; then
```

Comment:
maybe we can just use `test_port_ruby` if we know ruby is always installed, and document that we don't use nc because centos jenkins workers we use don't have it?

Reply:
We could, but`nc` is faster, as it doesn't incur the jruby startup slowdown

Reply:
got it. I thought of just pushing for a ruby script to try N times + sleep on each step, but I don't think there's much to gain from that. LGTM